src/systems/physics/Physics.cc

```diff
@@ -470,7 +470,8 @@ class gz::sim::systems::PhysicsPrivate
             physics::GetJointFromModel,
             physics::GetBasicJointProperties,
             physics::GetBasicJointState,
-            physics::SetBasicJointState>{};
+            physics::SetBasicJointState,
+            physics::SetMimicConstraintFeature>{};
```

Comment:
I think we shouldn't add this as a required part of `JointFeatureList` but rather handle it as an optional feature, like detachable joints

Reply:
made it an optional feature in https://github.com/gazebosim/gz-sim/pull/1838/commits/23c64dac0f13de21230d040fa87b5e90d1efd4fd

Reply:
fixed in https://github.com/gazebosim/gz-sim/pull/1838/commits/e1448b451d1c5f82e1ce8d0d1710a3e57eb00efd